README.md

```diff
@@ -94,58 +94,242 @@ git fetch upstream-code main
-- `letterpress` - The icon for the main area when all editors are closed. It's possible to provide different icons for `light` and `dark` themes.
-- `remoteIndicatorCommands` - The names of commands provided by the [`Eclipse Che Remote`](https://github.com/che-incubator/che-code/blob/main/code/extensions/che-remote/package.nls.json) extension.
-- `workbenchConfigFilePath` - The config file path. See an [example of the config file](https://github.com/che-incubator/che-code/blob/main/code/src/vs/code/browser/workbench/che/workbench-config.json).
-- `codiconCssFilePath` - The codicon css file path. Must contain CSS styles for `codicon`s. The content of the file is appended to [the coressponding css file](https://github.com/che-incubator/che-code/blob/main/code/src/vs/base/browser/ui/codicons/codicon/codicon.css). See an [example of the content](https://github.com/che-incubator/che-code/blob/803b864e8411bd57d617dabddfd8a132fac6c743/code/src/vs/base/browser/ui/codicons/codicon/codicon.css#L29-L33).
-
-NOTE:
--  All paths in the `branding/product.json` file must be relative to the `branding` folder. 
-For example, the `workbenchConfigFilePath` field might have the `anyFolder/myConfigFiles/workbench-config.json` value. 
-This means that the config file can be found by the path: `che-code/branding/anyFolder/myConfigFiles/workbench-config.json`
-- Currently, the [branding.sh](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) script is not run automatically when building this project. It needs to be integrated into the build process of the [downstream branded project or product](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/sync.sh#L96).
+
+6\. Run the `/branding/branding.sh` script. The [branding.sh](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) script searches for the branding resources in the [branding folder](https://github.com/che-incubator/che-code/tree/main/branding) and applies the changes.
+
+```
+$ ./branding/branding.sh
+```
+<!-- Currently, the [branding.sh](https://github.com/che-incubator/che-code/blob/main/branding/branding.sh) script is not run automatically when building this project. It needs to be integrated into the build process of the [downstream branded project or product](https://github.com/redhat-developer/devspaces-images/blob/devspaces-3-rhel-8/devspaces-code/build/scripts/sync.sh#L96). -->
+
+7\. Build the container image from the `/che-code/` directory and push the image to a container registry:
+
+```
+$ docker build -f build/dockerfiles/linux-musl.Dockerfile -t linux-musl-amd64 .
+
+$ docker build -f build/dockerfiles/linux-libc.Dockerfile -t linux-libc-amd64 .
+
+$ export DOCKER_BUILDKIT=1
+
+$ docker build -f build/dockerfiles/assembly.Dockerfile -t vs-code-open-source .
```

Comment:
I just noticed an inaccuracy in our examples:

the `next` tag is used in examples below, but by default docker sets `latest` tag at building images

so, please add `next` tag to the command
it should be:
`docker build -f build/dockerfiles/assembly.Dockerfile -t vs-code-open-source:next .`